packages/ui-lib/src/Form/molecules/ButtonsBundle.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import ButtonWithIcon from '../atoms/ButtonWithIcon';
+import Button from '../atoms/Button';
+import { IButtonProps } from '..';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  button {
+    text-wrap: nowrap;
+  }
+`;
+
+export type IButtonType = 'default' | 'icon-button'
+
+export interface IBButton extends IButtonProps {
```

Comment:
`IBButton` could we lengthen this name? It's a little confusing as this is Button Bundle but the B initial appears before Button and so looks a little like a typo.

(Maybe this changes anyway if we rename `ButtonsBundle`)